vmm/src/igvm/igvm_loader.rs

```diff
@@ -154,6 +155,15 @@ pub fn load_igvm(
     let mut gpas: Vec<GpaPages> = Vec::new();
     let proc_count = cpu_manager.lock().unwrap().vcpus().len() as u32;
 
+    #[cfg(feature = "sev_snp")]
+    let mut host_data_contents = [0; 32];
+    #[cfg(feature = "sev_snp")]
+    if !host_data.is_empty() {
+        assert_eq!(64, host_data.len());
```

Comment:
Assertions are for programming errors. If a valid `host_data` is expected to be exactly 64 chars, please enforce that as a part of `VmConfig::validate()`. Please also extend unit test `test_config_validation()` accordingly.

Reply:
Sounds good to me.

Reply:
@likebreath  How to run unit test with se_snp feature enabled?